controllers/discovery/initDiscoveryUserData.js

```diff
@@ -85,7 +85,8 @@ const InitDiscoveryUserData = async (req, res) => {
                 AND f2.user_id_followed = A.user_id
             ) AS is_followed,
             CommonUsers.common, 
-            B.user_id_follower 
+            B.user_id_follower,
+		        (A.last_active_at > NOW() - INTERVAL '7 days') AS recently_active
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new field `recently_active` is a boolean that checks if the user has been active in the last 7 days. This seems fine, but it's important to ensure that the `last_active_at` field is always updated correctly when users perform actions on the platform. If not, this could lead to inaccurate results.

```diff
- (A.last_active_at > NOW() - INTERVAL '7 days') AS recently_active
+ (COALESCE(A.last_active_at, NOW()) > NOW() - INTERVAL '7 days') AS recently_active
```
This change ensures that if `last_active_at` is null, it will be replaced with the current timestamp, which means the user will be considered as not recently active.



<!-- This is an auto-generated comment by OSS CodeRabbit -->